src/renderer/features/wallets/ImportKeysModal/ImportKeysModal.tsx

```diff
@@ -0,0 +1,117 @@
+import { useUnit } from 'effector-react';
+import { useEffect } from 'react';
+
+import { Alert, BaseModal, Button, InfoLink, InputFile, InputHint } from '@renderer/shared/ui';
+import { useI18n } from '@renderer/app/providers';
+import { importKeysModel, TypedImportedDerivation } from '@renderer/entities/wallet';
+import { AccountId } from '@renderer/shared/core';
+import { cnTw } from '@renderer/shared/lib/utils';
+// @ts-ignore
+import templateFile from '@renderer/assets/files/dd-template.yaml';
+
+type Props = {
+  isOpen: boolean;
+  rootAccountId: AccountId;
+  existingKeys: TypedImportedDerivation[];
+  onClose: () => void;
+  onConfirm: () => void;
+};
+
+export const ImportKeysModal = ({ isOpen, rootAccountId, existingKeys, onClose, onConfirm }: Props) => {
+  const { t } = useI18n();
+  const validationError = useUnit(importKeysModel.$validationError);
+  const successReport = useUnit(importKeysModel.$successReport);
+
+  useEffect(() => {
+    importKeysModel.events.importStarted({ derivations: existingKeys, root: rootAccountId });
```

Comment:
`importStarted` name states that I've made some meaningful action, in reality I want to init clean model

Reply:
renamed to `resetValues`